### PR TITLE
Add overview and running instructions

### PR DESCRIPTION
**Pull request by cpurta**
Added in the rest of the requirements in #1 which includes instructions to running the server and also a overview of what the code does. 

numer.ai username: cpurta

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,116 @@ For production, the API sever requires the following environment variables
 -   `PORT`
 -   `API_KEY`
```
[the remark this answers is not in the document]
Comment (cpurta) [reply README.md]: That's for a production server, correct? If so should the example sourcing script not have those env vars?

[the remark this answers is not in the document]
Comment (cpurta) [reply README.md]: Or are you talking about removing the entire section preceding the `Overview` section?

[the remark this answers is not in the document]
Comment (philipcmonk) [reply README.md]: Yes, I mean that you describe everything before the `Overview` section more clearly below, don't you?

**Comment by cpurta** [comment]
Yeah. It took me a minute to understand what you were saying. Cleaned everything up and removed everything before my changes.

**Comment by philipcmonk** [comment]
Awesome!  Your additional 8 NMR should be in your account now.

**Pull request merged**